packs/feats/feral-scramble.json

```diff
@@ -11,7 +11,7 @@
         },
         "category": "class",
         "description": {
-            "value": "<p><strong>Requirements</strong> You're in hybrid or animal shape.</p>\n<hr />\n<p>Sinking your claws in, you propel yourself at shocking speed. You Climb up to your land Speed and make a claw Strike at any point during that movement. After you finish the movement, attempt an Athletics check against the surface's Climb DC.</p>\n<p><strong>Success</strong> You maintain your position.</p>\n<p><strong>Failure</strong> You slide down 60 feet or to the ground without taking falling damage.</p>"
+            "value": "<p><strong>Requirements</strong> You're in hybrid or animal shape.</p>\n<hr />\n<p>Sinking your claws in, you propel yourself at shocking speed. You Climb up to your land Speed and make a claw Strike at any point during that movement. After you finish the movement, attempt an @Check[athletics]{Athletics} check against the surface's Climb DC.</p>\n<hr />\n<p><strong>Success</strong> You maintain your position.</p>\n<p><strong>Failure</strong> You slide down 60 feet or to the ground without taking falling damage.</p>"
```

Comment:
Remove {Athletics} from the check. It's aliasing it back to the same name.

Reply:
done.